cmake/projects/x264/hunter.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2017 Alexandre Pretyman
+# All rights reserved.
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_configuration_types)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    x264
+    VERSION
+    "snapshot-20170420-2245"
+    URL
+    "ftp://ftp.videolan.org/pub/x264/snapshots/x264-snapshot-20170420-2245.tar.bz2"
+    SHA1
+    679b8c4f39f375090bf64e5c80c83303e4ddc37c
+)
+
+hunter_configuration_types(x264 CONFIGURATION_TYPES Release)
+hunter_pick_scheme(DEFAULT url_sha1_autotools)
+hunter_cacheable(x264)
+hunter_cmake_args(
+    x264
+    CMAKE_ARGS         # do not use double quotes on CMAKE_ARGS
```

Comment:
> "PKGCONFIG_EXPORT_TARGETS=x264;other-pkg-config-module2;other-pkg-config-module3"

Should work

> PKGCONFIG_EXPORT_TARGETS="x264;other-pkg-config-module2;other-pkg-config-module3"

Yes, because it's unquoted argument

> We tried before ;-)

Yep, I remember. This is the reason I have this section:
* http://cgold.readthedocs.io/en/latest/tutorials/variables/regular.html#quotes